src/layout/section/columnedContentSection/ColumnedContentSection.story.mdx

```diff
@@ -0,0 +1,182 @@
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs/blocks'
+
+import { CheckShieldIcon } from '../../../icon/checkShieldIcon'
+import { DirectionIcon } from '../../../icon/directionIcon'
+import { IdCardVerifiedIcon } from '../../../icon/idCardVerifiedIcon'
+import { ColumnedContentSection, ColumnedSectionContentMediaKind } from './index'
+
+<Meta title="Sections|ColumnedContentSection" />
+
+# **ColumnedContentSection**
+
+<Preview>
+  <Story name="Simple">
+    <ColumnedContentSection
+      title="Allez où vous voulez. D'où vous voulez."
+      topLinkLabel="En savoir plus"
+      topLinkHref="https://www.blablacar.fr/how-does-car-sharing-work"
+      columnContentList={[
+        {
+          title: 'Pratique',
+          content:
+            'Trouvez rapidement un covoiturage ou un bus à proximité parmi les millions de trajets proposés.',
+        },
+        {
+          title: 'Facile',
+          content:
+            'Trouvez le trajet parfait ! Il vous suffit d’entrer votre adresse exacte, de choisir le voyage qui vous convient le mieux, et de réserver.',
+        },
+        {
+          title: 'Direct',
+          content:
+            'Que vous prévoyiez à l’avance ou réserviez en dernière minute, vous trouverez toujours un tarif avantageux.',
+        },
+      ]}
+    />
+  </Story>
+  <Story name="Blog">
```

Comment:
Can we have one Story by Preview, plz

Reply:
We can't, due to the storybook 5.3 failure: it renders 4 times the same story.

<img width="1020" alt="Capture d’écran 2020-11-03 à 11 02 22" src="https://user-images.githubusercontent.com/729186/97971924-1222d400-1dc4-11eb-9ccf-301f8d62ec54.png">

Reply:
Removed the `<Story>` to have multiple `<Preview>`